packages/documentation/src/stories/internet-header/components/footer/footer.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { Args, Meta, StoryContext, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { BADGE } from '../../../../../.storybook/constants';
+import customFooterConfig from './custom-config/custom-footer-config';
+import { getAttributes } from '../../../../utils';
+import { spread } from '@open-wc/lit-helpers';
+
+const meta: Meta = {
+  title: 'Internet Header/Footer',
+  component: 'swisspost-internet-footer',
+  render: renderInternetFooter,
+  decorators: [hiddenHeader],
+  parameters: {
+    layout: 'fullscreen',
+    badges: [BADGE.STABLE],
+  },
+  argTypes: {
+    customConfig: {
+      control: 'object',
+      table: {
+        type: {
+          summary: 'ICustomConfig',
+          detail: JSON.stringify(customFooterConfig),
+        },
+      },
+    },
+  },
+};
+
+export default meta;
+
+// DECORATORS
+function hiddenHeader(story: any, { args }: StoryContext) {
+  return html`
+    <style>
+      swisspost-internet-header {
+        display: none;
+      }
+    </style>
+    ${story()}
+  `;
+}
+
+// RENDERER
+function renderInternetFooter(args: Args) {
+  const props = args.customConfig ? { 'custom-config': JSON.stringify(args.customConfig) } : {};
+  return html`
+    <swisspost-internet-header ${spread(props)} project="test"></swisspost-internet-header>
+    <swisspost-internet-footer></swisspost-internet-footer>
+  `;
+}
```

Comment:
This implementation differs a lot from the breadcrumbs stories file, which in my opinion is the better implementation.